common/src/main/resources/db/changelog/v2021/add_coverage_delta.sql

```diff
@@ -0,0 +1,17 @@
+--liquibase formatted sql
+--  -------------------------------------------------------------------------------------------------------------------
+
+--changeset enolan:add_coverage_delta_table failOnError:true
+CREATE TABLE coverage_delta (
+    id BIGSERIAL NOT NULL,
+    bene_coverage_period_id INTEGER NOT NULL,
+    beneficiary_id VARCHAR(255) NOT NULL,
+    type VARCHAR(32) NOT NULL, -- ADDED/REMOVED
+    created timestamp
+);
```

Comment:
Do we want to track which search it is related to explicitly? Right now we can't tie it to a specific CoverageSearchEvent?

Reply:
I thought that the created timestamp would be sufficient for analysis.  If there is a justifying use case, it can always be added.